packages/component-header/src/components/HeaderMain/NavbarContainer/index.js

```diff
@@ -38,11 +38,13 @@ const NavbarContainer = () => {
     // @ts-ignore
     <Wrapper breakpoint={breakpoint} data-testid="navigation" aria-label="Main">
       <div className="content-container">
```

Comment:
Navigation should take over entire screen if the navigation has links or buttons, 
exclude `content-container` if empty will allow smaller nav. 
Something like the following should work, 
```
{ (
mobileNavTree.length && isMobile ||
navTree.length && !isMobile ||
buttons.length
) && (
<div className="content-container"></div>
) }
```

Reply:
@scott-williams-az resolved, and ready for new review